modules/ROOT/pages/customize-nic.adoc

```diff
@@ -0,0 +1,48 @@
+:experimental:
+= How to Customize a NIC Name
+
+== Using a `systemd` Link File
+You can create a `systemd` xref:https://www.freedesktop.org/software/systemd/man/systemd.link.html[link file] with `ignition` configs.
```

Comment:
`s/ignition/Ignition`

Reply:
For your reference: https://github.com/coreos/fedora-coreos-config/pull/344#discussion_r435419740

Reply:
thanks!